piton/ariane_setup.sh

```diff
@@ -76,7 +76,11 @@ export ARIANE_ROOT=${PITON_ROOT}/piton/design/chip/tile/ariane/
 ## GCC and RISCV GCC setup
 export CXX=g++ CC=gcc
 # customize this to a fast local disk
-export RISCV=/scratch/`whoami`/riscv_install
+
+if [ "$RISCV" ==  "" ]
```

Comment:
So one concern I have about `$RISCV` is that ariane_build_tools is going to try to overwrite the contents of whatever that directory is. Do we maybe want to print a message in the else case of this to warn people of that somehow?

Also should we change that /scratch/ location maybe? Just put it under `$ARIANE_ROOT/tmp` like the other stuff?

Reply:
I would not like to be inside the repo, because often times I have 2/3/4 openpiton repos pointing to the same RISCV variable.

Reply:
Regarding the warning, we could change the variable to be $RISCV_INSTALLATION, or something that would not conflict with an existing variable. The warning is fine too

Reply:
Personally I'd prefer a location that didn't require root privileges because it encourages people to run our scripts under sudo (I've seen this several times now). Maybe we could go for `$HOME/scratch/riscv_install`?